docker/rucio_client/scripts/updateDDMQuota

```diff
@@ -2,54 +2,36 @@
 
 import math
 from rucio.client import Client
+from tabulate import tabulate
```

Comment:
Do we have tabulate installed in the container already?

Reply:
yes
```
[haozturk@lxplus9115 docker]$ podman run -d --name rucio-cron registry.cern.ch/cmsrucio/rucio_client:release-33.6.1.cms7
ERRO[0000] cannot find UID/GID for user haozturk: no subuid ranges found for user "haozturk" in /etc/subuid - check rootless mode in man pages. 
WARN[0000] Using rootless single mapping into the namespace. This might break some images. Check /etc/subuid and /etc/subgid for adding sub*ids if not using a network user 
Trying to pull registry.cern.ch/cmsrucio/rucio_client:release-33.6.1.cms7...
Getting image source signatures
Copying blob 0a1c263165d7 done  
Copying blob 45f7c147f34e done  
Copying config f485067934 done  
Writing manifest to image destination
1cefce93dd2a98c63c48b122acb7cfce566e89d3bf72c7e03334d9a0074449f8
[haozturk@lxplus9115 docker]$ podman ps
CONTAINER ID  IMAGE                                                       COMMAND     CREATED             STATUS             PORTS       NAMES
1cefce93dd2a  registry.cern.ch/cmsrucio/rucio_client:release-33.6.1.cms7              About a minute ago  Up About a minute              rucio-cron
[haozturk@lxplus9115 docker]$ podman exec -it rucio-cron /bin/bash
[root@1cefce93dd2a ~]# 
[root@1cefce93dd2a ~]# python3
Python 3.9.18 (main, Jan  4 2024, 00:00:00) 
[GCC 11.4.1 20230605 (Red Hat 11.4.1-2)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from tabulate import tabulate
```